### ISSUE TITLE
Azure Maps SDK for Unity?

### ISSUE THREAD

**Issue by zeedyBOI** [Azure/azure-sdk-for-net#44214]
### Type of issue

Missing information

### Description

I have been developing an application on Unity with big focus on the 3D Map the the Bing Map SDK offered.
Since the SDK/service is being discontinued next year and no information was given regarding this SDK's specific platform, I am left wondering if this SDK, soon to be dead, with no real alternative from Microsoft,  even tho similar services for different platform are being migrated to Azure ecosystem, is also being migrated eventually or will be completely dead by June 2025?

TLDR: will the Unity SDK be migrated to Azure ecosystem or not?

### Page URL

https://learn.microsoft.com/en-us/dotnet/api/overview/azure/maps?view=azure-dotnet

### Content source URL

https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/api/overview/azure/latest/maps.md

### Document Version Independent Id

0db9ebc4-c941-e80d-08ad-c16d844bbdd1

### Article author

@azure-sdk

### Metadata

* ID: d8649965-8ca8-9dff-d9b8-33a3d03d7426 
* Service: **maps**

**Comment by dubiety** [comment]
Hi @zeedyBOI ,

My apologies, but we don't have any plan to support Unity SDK in Azure Maps. The alternative is to send the request from Unity directly.

**Comment by dubiety** [comment]
Close the issue since no any follow up work item.
